oshdb-filter/src/main/java/org/heigit/ohsome/oshdb/filter/Filter.java

```diff
@@ -100,4 +99,8 @@ public boolean applyOSMGeometry(OSMEntity entity, Supplier<Geometry> geometrySup
       }
     });
   }
+
+  private Filter() {
+    // utility class
+  }
```

Comment:
is this really needed for an abstract class?

Reply:
No I don't think so, the private constructor is enough, like it is e.ge `OSHEntityTimeUtils`. Even sonarlint was complaining why I don't do :joy:

Reply:
I meant, that the Filter should better not be abstract.

Reply:
> why I don't do
you mean convert the class to an interface?

Reply:
You are right, that is what sonarlint suggest ("because it is abstract class with only static methods"), but actually what it should be is [rule-1118 Utility Class](https://rules.sonarsource.com/java/RSPEC-1118)
So wrong here is that this class is abstract.